### PR TITLE
Small fixes for the Qt5 port

### PR DESCRIPTION
**Pull request by City-busz**
This makes it possible to build fcitx-libpinyin properly with Qt5.

### CODE REVIEW

File: dictmanager/importer.h [hunk 1]
```diff
@@ -22,7 +22,7 @@
 #define FCITX_IMPORTER_H
 
 #include <QString>
-#include <fcitx-qt/fcitxqtconnection.h>
```
Comment (wengxt) [inline dictmanager/importer.h]: this is wrong actually, the prefix is not needed.

File: dictmanager/dictmanager.h [hunk 2]
```diff
@@ -22,7 +22,7 @@
 #define FCITX_DICTMANAGER_H
 
 #include <QMainWindow>
-#include <fcitx-qt/fcitxqtconfiguiwidget.h>
+#include <FcitxQt5/FcitxQtWidgetsAddons/fcitxqtconfiguiwidget.h>
```
Comment (wengxt) [inline dictmanager/dictmanager.h]: same to this.

File: dictmanager/importer.h (hunk 1 above)
Comment (wengxt) [inline dictmanager/importer.h]: I mean #include <fcitxqtconnection.h> is what it supposed to be. without <FcitxQt5...

**Comment by City-busz** [comment]
Okay. I updated my previous commit.

**Pull request merged**